### ISSUE TITLE
Disable GitHub issue tracker?

### ISSUE THREAD

**Issue by rhencke** [wildfly/wildfly#10310]
As Wildfly uses JIRA instead of Github issues, would it make sense to disable the issue tracker for this project?  This can be done in the Settings tab of this repository, for owners.

(I thought about filing this on the Wildfly JIRA instead, but the irony of filing the issue here instead was too tempting)

**Comment by ctomc** [comment]
Yeah, that is something we definitely want to do but it breaks some CI / GitHub integration we have in place.
GitHub exposes each pull request as issue as well, but for some operations, you need to use issues api for pull requests instead of going "full pull request" api.
When we disable issue tracker on repository those rest api calls fail.
At least that was a while back when we ware latest looking into it. It could be that we can now disable it safely.

**Comment by rhusar** [comment]
@ctomc Would it be possible to give it a try again? It's been a while.

**Comment by ctomc** [comment]
@rhusar It should be fine at this point. we just need one of the @wildfly/mergers-full to change settings.

**Comment by rhusar** [comment]
@ctomc Perfect! Though, I am afraid its not enough to be @wildfly/mergers-full but you need to be organization owner.
